SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/SlaveMethods/BaseContinuousSendMethod.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Plugin.Microsoft.Azure.SignalR.Benchmark.SlaveMethods
+{
+    public class BaseContinuousSendMethod
+    {
+        protected async Task ContinuousSend<T, TKey, TValue>(
+        T connection, IDictionary<TKey, TValue> data, 
+        Func<T, IDictionary<TKey, TValue>, Task> f, 
+        TimeSpan duration, TimeSpan interval)
+        {
+            // Random delay in [1, interval) ms
+            var rand = new Random(DateTime.Now.Millisecond);
```

Comment:
The default constructor random do the similar thing: `new Random()` == `new Random(DateTime.Now.Tick)`.

Reply:
Got it, thanks.